taipy/core/scenario/scenario.py

```diff
@@ -585,15 +585,15 @@ def submit(
     def export(
         self,
         folder_path: Union[str, pathlib.Path],
-        override: Optional[bool] = False,
-        include_data: Optional[bool] = False,
+        override: bool = False,
+        include_data: bool = False,
     ):
         """Export all related entities of this scenario to a folder.
 
         Parameters:
             folder_path (Union[str, pathlib.Path]): The folder path to export the scenario to.
```

Comment:
```suggestion
            folder_path (Union[str, pathlib.Path]): The folder path to export the scenario to.
                The path must not exist.
```

Reply:
What do you think about
```suggestion
            folder_path (Union[str, pathlib.Path]): The folder path to export the scenario to.
                If the path exists and the override parameter is False, an exception is raised.
```

The path can exist, as long as the `override` is True

Reply:
Much better.